crux-test/test/crux/fixtures/standalone.clj

```diff
@@ -1,16 +1,11 @@
 (ns crux.fixtures.standalone
-  (:require [crux.fixtures.api :as fapi]
+  (:require [crux.fixtures :as fix]
```

Comment:
I would just kill this ns rather than deprecate.

Reply:
will do